app/index.js

```diff
@@ -1,11 +1,13 @@
 require('./insights').setup()
 require('log-timestamp')
 const { createServer } = require('./server')
+const { createAzuriteInfrastructure } = require('./storage/blob-storage')
```

Comment:
This is referenced in the `/storage/index` so you get it direct from there

```suggestion
const { createAzuriteInfrastructure } = require('./storage')
```